indexing-hadoop/src/main/java/io/druid/indexer/path/GranularUnprocessedPathSpec.java

```diff
@@ -74,13 +75,11 @@ public Job addInputPaths(HadoopDruidIndexerConfig config, Job job) throws IOExce
     final FileSystem fs = betaInput.getFileSystem(job.getConfiguration());
     final Granularity segmentGranularity = config.getGranularitySpec().getSegmentGranularity();
 
-    Map<Long, Long> inputModifiedTimes = new TreeMap<>(
-        Comparators.inverse(Comparators.comparable())
-    );
+    Map<Long, Long> inputModifiedTimes = new TreeMap<>(Ordering.natural());
```

Comment:
Reverted.

@drcrallen this order is used only in the body of the method, later the contents of this collection are converted and moved to another collection with different ordering. So I doubt testing is needed. Also I don't know *why* `inputModifiedTimes` should have reversed ordering, it dates back to "Initial commit of Druid OSS" in 2012 by @cheddar.